src/spot-client.ts

```diff
@@ -1,157 +1,153 @@
-import { AxiosRequestConfig } from 'axios';
-import { KlineInterval } from './types/shared';
-import { NewSpotOrder, OrderSide, OrderTypeSpot, SpotOrderQueryById } from './types/spot';
-import BaseRestClient from './util/BaseRestClient';
-import { GenericAPIResponse, getRestBaseUrl, RestClientOptions } from './util/requestUtils';
+import {AxiosRequestConfig} from 'axios';
+import {KlineInterval, APIResponse, SymbolInfo} from './types/shared';
+import {NewSpotOrder, OrderSide, OrderTypeSpot, SpotOrderQueryById} from './types/spot';
+import {GenericAPIResponse, getRestBaseUrl, RestClientOptions} from './util/requestUtils';
```

Comment:
Lots of style changes here, please try to preserve the existing styling (2-spaced indent, 1 space around the braces)